src/api/app/jobs/consistency_check.rb

```diff
@@ -140,9 +140,11 @@ def import_project_from_backend(project)
     meta = Backend::Connection.get("/source/#{project}/_meta").body
     project = Project.new(name: project)
     project.commit_opts = {no_backend_write: 1}
-    project.update_from_xml(Xmlhash.parse(meta))
+    project.update_from_xml!(Xmlhash.parse(meta))
     project.save!
     return ""
+  rescue APIException
+    # FIXME: Check if we want to do something in this case
```

Comment:
This keep exactly the same behaviour here, but I am not sure if we want to do something in this case. Returning an error message for example. :thinking: